presto-docs/src/main/sphinx/release/release-0.224.rst

```diff
@@ -0,0 +1,38 @@
+=============
+Release 0.224
+=============
+
+General Changes
+---------------
+
+* Fix mis-swapped CPU time and scheduled time in query plan.
+* Fix correctness bug on queries containing nested coercible type cast since release 0.220.
```

Comment:
```
Fix correctness issue for queries containing nested coercible type cast.
```

Reply:
Do we need to say `Fix a correctness issue`?

Reply:
I think `Fix a correctness issue` is better, although we don't have consistency (we have both) in our past release notes.

Reply:
I do think it's valuable to say when the correctness bug was introduced so people know whether they were vulnerable depending on what version they are upgrading from.

Reply:
@rschlussel Sorry I should clarify my question, I'm basically asking if we should choose `fix a correctness issue` instead of `fix correctness issue`.

Reply:
Either?  `Fix a correctness issue` is better English, but our release notes tend to have a more shorthand form sometimes (e.g. `add configuration property` rather than `add a configuration property`), so I don't think it really matters.